project/ISSUE-TRIAGE.md

```diff
@@ -31,53 +28,36 @@ If the author provides the standard information but you are still unable to tria
-the template to the issue's description also add any required labels to the issue for the classification and difficulty.
-
-Here is a sample summary for an [issue](https://github.com/docker/docker/issues/10545).
-
-```
-**Summary**: docker rm can return a non-zero exit code if the container does not
-exist and it is not easy to parse the error message.
-
-**Proposed solution**:
+### 2. Classify the Issue
 
-docker rm should have consistent exit codes for different types of errors so
-that the user can easily script and know the reason why the command failed. 
+An issue can have multiple of the following labels.
 
-```
-
-### Step 3: Classify the Issue
-
-Classifications help both to inform readers about an issue's priority and how to resolve it.
-This is also helpful for identifying new, critical issues.  "Kinds of" are
-applied to the issue or pull request using labels.  You can apply one or more labels.
-
-
-Kinds of classifications:
+#### Issue kind
 
 | Kind             | Description                                                                                                                     |
 |------------------|---------------------------------------------------------------------------------------------------------------------------------|
-| kind/enhancement | Enhancement are not bugs or new features but can drastically improve usability or performance of a project component.           |
-| kind/cleanup     | Refactoring code or otherwise clarifying documentation.                                                                         |
-| kind/content     | Content that is not documentation such as help or error messages.                                                               |
-| kind/graphics    | Work involving graphics skill                                                                                                   |
-| kind/regression  | Regressions are usually easy fixes as hopefully the action worked previously and git history can be used to propose a solution. |
 | kind/bug         | Bugs are bugs. The cause may or may not be known at triage time so debugging should be taken account into the time estimate.    |
+| kind/enhancement | Enhancement are not bugs or new features but can drastically improve usability or performance of a project component.           |
 | kind/feature     | Functionality or other elements that the project does not currently support.  Features are new and shinny.                      |
-| kind/question    | Contains a user or contributor question requiring a response.                                                                   |
-| kind/usecase     | A description of a user or contributor situation requiring a response perhaps in code or documentation.                         |
 | kind/writing     | Writing documentation, man pages, articles, blogs, or other significant word-driven task.                                       |
```

Comment:
Makes sense. I suggest `kind/docs` to say that this is about writing docs, but `area/` should probably remain a functional area (so maybe `kind/docs area/api` if it's about API documentation?).